src/Serializer/ItemNormalizer.php

```diff
@@ -86,11 +86,7 @@ private function updateObjectToPopulate(array $data, array &$context): void
 
     private function getContextUriVariables(array $data, $operation, array $context): array
     {
-        if (!isset($context['uri_variables'])) {
-            return ['id' => $data['id']];
-        }
-
-        $uriVariables = $context['uri_variables'];
+        $uriVariables = $context['uri_variables'] ?? $data;
```

Comment:
@Aerendir can you try this?

Reply:
Hi @soyuka , I've tried the fix, but it doesn't work...

In `ItemNormalizer::getContextUriVariables()`, there is no key `$context['uri_variables']`, so the variable `$uriVariable` gets the value of `$data`, but `$data` contains the entity and so there will never be the uri variables needed.

I'm going to reproduce in tests what's going on...